File: test_rls.js
require('dotenv').config();
const { Client } = require('pg');
const client = new Client({ connectionString: process.env.SUPABASE_URL });
async function run() {
  await client.connect();
  await client.query("ALTER TABLE ingestion_logs DISABLE ROW LEVEL SECURITY;");
  console.log("Disabled RLS on ingestion_logs");
  await client.end();
}
run().catch(console.error);

File: test_rls.py
import os
from dotenv import load_dotenv
load_dotenv()
import asyncio
from sqlalchemy import create_engine, text
engine = create_engine(os.getenv('SUPABASE_URL'))
with engine.connect() as conn:
    conn.execute(text('ALTER TABLE ingestion_logs DISABLE ROW LEVEL SECURITY;'))
    conn.commit()
    print('RLS Disabled')
